web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -1,37 +1,44 @@
 import { TypeValue, UrlList, UrlType } from 'lib/rdf/types';
 
+/** Anchored metadata AKA "Additional Info" - Editable only with a signed Ledger TX. */
 export interface ProjectMetadataLD {
   '@context': Context;
-  '@type': string;
+  '@type': string; // regen:C01-Project
   'schema:name': string;
-  'schema:image': UrlType;
-  'schema:creditText': string;
-  'schema:description'?: string;
   'schema:location': any;
   'regen:projectType': string;
   'regen:projectActivity': ProjectActivity;
-  'regen:projectDeveloper': ProjectStakeholder;
-  'regen:landSteward': ProjectStakeholder;
-  'regen:landOwner': ProjectStakeholder;
-  'regen:projectOriginator': ProjectStakeholder;
   'regen:offsetGenerationMethod': string;
   // TODO: not sure why regen:offsetGenerationMethod isn't compacting as expected so added this:
   'http://regen.network/offsetGenerationMethod'?: string;
   'regen:projectSize': ProjectSize;
   'regen:projectStartDate': TypeValue;
   'regen:projectEndDate': TypeValue;
-  'regen:boundaries': TypeValue;
-  'regen:creditClass': CreditClass;
-  'regen:landManagementActions': LandManagementActions;
+}
+
+/** Un-anchored metadata from our DB. This is editable without a Ledger TX. */
+export interface ProjectPageMetadataLD {
+  '@context': Context;
+  '@type': string; // regen:C01-Project-Page
+  'schema:image': UrlType;
+  'schema:creditText': string;
   'regen:galleryPhotos': UrlList;
```

Comment:
note: some C01 in the ops repo don't have `galleryPhotos` or `previewPhoto`. do we need to mark those as optional in this interface? i.e. https://github.com/regen-network/regen-registry-standards/blob/main/ops/C01/project-metadata/C01-ProjectId1218-metadata.jsonld

Reply:
Good point. We are using `Partial<T>` to make each field optional in code right now. I am going to keep thinking about whether it would be better to explicitly set optional fields here.

Reply:
I think it would best to make it explicit since we do know which fields are required and which ones are optional
for instance, we always have some `@context` and `@type`
using `Partial<ProjectPageMetadataLD>` allows those fields to not be there...

Reply:
Using explicit optionals now.